### PR TITLE
Update PPP - Zong Han

### PR DESCRIPTION
**Pull request by sharpstorm**
## Changes
This PR updates the PPP for Zong Han to the latest `v1.3b` milestone.

## Related Issues
None

### CODE REVIEW

File: docs/team/sharpstorm.md [hunk 1]
```diff
@@ -5,50 +5,64 @@ title: Zong Han's Project Portfolio Page
...
+    * Added implementation and architectural design details for Date Time Input Parsing
+    * Added implementation details for UI result display mode switching
+    * Added use cases for Appointments
 
 * **Community**:
-  * `to be added soon`
-
-* **Tools**:
-  * `to be added soon`
+  * Reviewed a significant amount of PRs within the team ([347 comments](https://nus-cs2103-ay2122s2.github.io/dashboards/contents/tp-comments.html) across [60+ PRs](https://github.com/AY2122S2-CS2103-W17-1/tp/pulls?q=is%3Apr+is%3Aclosed))
```
Comment (dandaandaaaaaan) [inline docs/team/sharpstorm.md:67]: I suggest for the across PRs that part, can use this filter instead
https://github.com/AY2122S2-CS2103-W17-1/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3Asharpstorm

it'll show what PRs u reviewed in particular

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm, one minor suggestion on the link, thanks for making the format. i'll follow this roughly for mine as well

File: docs/team/sharpstorm.md (hunk 1 above)
Comment (sharpstorm) [reply docs/team/sharpstorm.md]: Thanks, adopted in commit [1f0c97c](https://github.com/AY2122S2-CS2103-W17-1/tp/pull/317/commits/1f0c97c0db655a29be08cbf4049fe7f7a58f0945).

File: docs/team/sharpstorm.md [hunk 2]
```diff
@@ -5,50 +5,64 @@ title: Zong Han's Project Portfolio Page
...
+  * Built the backbone for the Appointments subsystem, including code for each of the `Storage`, `Ui`, `Logic` and `Model` components.
+  * Restructured and Unified the UI result display implementation to allow different types of models to be displayed in the UI using a single system.
+  * Test cases for the Appointments subsystem, as well as overall test case maintenance of the `model` package.
 
 * **Project management**:
-  * Served as the overall project lead in organizing meetings and working with the entire team to ensure that integration of components proceed smoothly.
-  * Served as the project schedule manager in reminding team members of deadlines, and setting internal deadlines to ensure that the project milestone deadline can be met.
+  * Served as the **overall project lead** in organizing meetings and working with the entire team to ensure that integration of components proceed smoothly.
+  * Served as the **project schedule manager** to manage internal deadlines across all team members to ensure that the project milestone deadlines can be met.
+  * Upkeep of code quality and coverage across all members
```
[the remark this answers is not in the document]
Comment (sharpstorm) [reply docs/team/sharpstorm.md]: Thanks for spotting it, fixed in commit [1f0c97c](https://github.com/AY2122S2-CS2103-W17-1/tp/pull/317/commits/1f0c97c0db655a29be08cbf4049fe7f7a58f0945).

**Review by Wei-Xinn — APPROVED** [review]
lgtm

**Pull request merged**